ENIGMAsystem/SHELL/Universal_System/Object_Tiers/collisions_object.h

```diff
@@ -52,8 +52,8 @@ namespace enigma
         int $bbox_right()  const;
         int $bbox_top()    const;
         int $bbox_bottom() const;
-        const bbox_rect_t& $bbox_relative() const;
-        const bbox_rect_t& $bbox() const;
+        const BoundingBox $bbox_relative() const;
```

Comment:
Might be better to make this const-ref.

Reply:
refrence to what? bbox relative isnt stored anymore

Reply:
This comment was supposed to be pointing to $bbox; is that gone, now?

Reply:
I don't think it returns a reference anymore it seems odd that one would return a reference an the other wouldn't